### ISSUE TITLE
Assertion failure

### ISSUE THREAD

**Issue by grasshoppermouse** [EddyVerbruggen/nativescript-calendar#10]
Everything works great until I try and add an event to a new calendar, at which point I get:

`*** Assertion failure in +[EKCalendar calendarForEntityType:eventStore:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/EventKit_Sim/EventKit-672/EventKit/iOS/src/EKCalendar.m:52`

Here is my code:

```
  toggleFavorite(e) {

      var options = {
          calendar: {name: "AAPA 2017"}, \\ Works fine until I add this line
          title: this.session.title,
          startDate: new Date(new Date().getTime() + (60 * 60 * 1000)),
          endDate: new Date(new Date().getTime() + (2 * 60 * 60 * 1000)),
          location: this.session.location,
          reminders: {first: 10},
          id: "0"
      };

      let a = this.session
      if (!this.session.favorite) {

          Calendar.createEvent(options).then(
              function(createdId) {
                  console.log("Created Event with ID: " + createdId);
                  a.cal_id = createdId;
                  a.favorite = true;
              },
              function(error) {
                  console.log("Error creating an Event: " + error);
              }
          );

      } else {

          options.id = this.session.cal_id

          Calendar.deleteEvents(options).then(
              function(deletedEventIds) {
                  console.log('Event deleted');
                  a.cal_id = "0";
                  a.favorite = false;
              },
              function(error) {
                  console.log("Error deleting Events: " + error);
              }
          )
      }
  }
```

**Comment by EddyVerbruggen** [comment]
Could it be the space in the Calendar name?

**Comment by grasshoppermouse** [comment]
Changed the name to `aapa`. Got the same error:

```
Oct 29 07:23:46 iMac aapaconferenceapp[3927]: *** Assertion failure in +[EKCalendar calendarForEntityType:eventStore:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/EventKit_Sim/EventKit-672/EventKit/iOS/src/EKCalendar.m:52
Oct 29 07:23:46 iMac com.apple.CoreSimulator.SimDevice.1B06A134-C3EC-4578-A0F7-F4A3423B4A6E.launchd_sim[1561] (UIKitApplication:org.nativescript.aapaconferenceapp[0x3caf][3927]): Service exited due to Segmentation fault: 11
```

**Comment by EddyVerbruggen** [comment]
Thanks for testing that, I'll take a look!

**Comment by EddyVerbruggen** [comment]
Am I right concluding this test was in the Simulator? On my sim it crashes with the same error, on an actual device all is fine (even with the space in the Calendar name).

**Comment by grasshoppermouse** [comment]
Yes, I'm developing in the Simulator. I haven't tried it on an actual device yet.

**Comment by EddyVerbruggen** [comment]
Thanks for confirming that so quickly - I've found the issue that pops up on Sim but not on a real device (which is a bit weird). Fixed in 1.1.3.
